functests/4_latency/latency.go

```diff
@@ -428,6 +434,17 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 	err = pods.WaitForPhase(testPod, corev1.PodRunning, 2*time.Minute)
 	Expect(err).ToNot(HaveOccurred())
 
+	if runtime, _ := strconv.Atoi(latencyTestRuntime); runtime > 1 {
+		By("Checking actual CPUs number for the running pod")
+		limitsCpusQuantity := testPod.Spec.Containers[0].Resources.Limits.Cpu()
+		RequestsCpusQuantity := testPod.Spec.Containers[0].Resources.Requests.Cpu()
+		//letancy pod is guaranteed
+		//CmpInt64 returns 0 if the quantity is equal to y, -1 if the quantity is less than y, or 1 if the quantity is greater than y
+		//func (q *Quantity) CmpInt64(y int64) int{..}
+		Expect(limitsCpusQuantity.CmpInt64(int64(latencyTestCpus))).To(Equal(0), fmt.Sprintf("actual limits of cpus number used for the latency pod is not as set in LATENCY_TEST_CPUS, actual number is: %s", limitsCpusQuantity))
```

Comment:
I'd use
```
func isEqual(qty *resource.Quantity, int amount) bool {
    return qty.CmpInt64(int64(amount)) == 0
}
```
note the pointer only because this is what `Cpu()` returns :\

and then
```
Expect(isEqual(limitsCpusQuantity, latencyTestCpus).To(BeTrue(), "actual limits of cpus number used for the latency pod is not as set in LATENCY_TEST_CPUS, actual number is: %s", limitsCpusQuantity)
```
(same for `Requests`)

Reply:
sounds better to me, modified it. Thank you!